tiledb/sm/fragment/fragment_metadata.cc

```diff
@@ -4559,189 +4497,140 @@ Status FragmentMetadata::write_tile_validity_offsets(
 
 Status FragmentMetadata::store_tile_mins(
     unsigned idx, const EncryptionKey& encryption_key, uint64_t* nbytes) {
```

Comment:
This can return void.

Reply:
Please do for all other store_ function touched in this review.

Reply:
changed to return void